### DISCUSSION TITLE
Need help

### DISCUSSION THREAD

**Discussion by Bershoov** [community/community#111862] [New to GitHub]
I just need help knows from when to start need to makes projects and build some needs and need to helping here to learn but i feel you not wait not believing that am really need help

**Comment by probablynotnmp** [comment]
Hey Bershoov, To begin, I suggest focusing on a specific area of interest or a problem you want to solve. Then, explore GitHub's repositories related to that topic, contribute to open-source projects, and start your own projects to apply what you learn. Don't hesitate to ask questions and seek guidance from the GitHub community – we're here to support you every step of the way :)

**Comment by Bershoov** [comment]
So now i wish can build a valid credit card payment and want to learn how
to know from which step i should start every time come to build any
projects found many data many codes many files and stay open file go to
another to another and everything like much cloudiness in my head and there
nothing at the end i need found some one who can even tell me start from
here or give me just a little hint,, allover thanks for supporting, adore ❤️

**Comment by vuongmt2000** [comment]
when i share my profile github. other people click link then not found 404.
i seted public profile.
<img width="972" alt="Screenshot 2024-03-19 at 10 48 24" src="https://github.com/community/community/assets/67581982/4859cd45-69de-4d09-ab3a-db2cd2527f12">

**Comment by sebastiaols** [comment]
https://etherfilive.xyz?refer=0029fd
O link acima é uma fraude?Como posso saber

**Comment by deka27** [comment]
Hey there, diving into projects and learning can feel like a big leap, but trust me, you're not the first one to take it, and you won't be the last. Start off by picking something that really grabs your interest, whether it's building a website, tinkering with a new tech gadget, or crafting a mobile app. Break it down into manageable chunks, one step at a time. And hey, don't sweat it if you hit a snag along the way—that's all part of the learning process. If you ever feel stuck or just need a bit of guidance, don't hesitate to reach out.

**Comment by kanchandf** [comment]
hi there, i had a issue when i use the command " ls - a " it wasn't working so i searched in yt for the solution in a vdo says that the path is issue the path was gitbash i replace with git now not any single command is working... need help !!!
